nautobot/cloud/views.py

```diff
@@ -122,31 +126,84 @@ class CloudResourceTypeUIViewSet(NautobotUIViewSet):
+                weight=1000,
+                tab_id="services",
+                label="Cloud Services",
+                url_name="cloud:cloudresourcetype_services",
+                related_object_attribute="cloud_services",
+            ),
+        ),
+    )
 
-            services = instance.cloud_services.restrict(request.user, "view")
-            services_table = CloudServiceTable(services)
-            services_table.columns.hide("cloud_resource_type")
-            RequestConfig(request, paginate).configure(services_table)
+    def get_extra_context(self, request, instance):
+        context = super().get_extra_context(request, instance)
+        context.update({"object_detail_content": self.object_detail_content})
+        return context
 
-            context.update(
-                {
-                    "networks_count": networks.count(),
-                    "networks_table": networks_table,
-                    "services_count": services.count(),
-                    "services_table": services_table,
-                }
-            )
+    @action(detail=True, url_path="networks")
+    def networks(self, request, *args, **kwargs):
+        instance = self.get_object()
+        networks = instance.cloud_networks.restrict(request.user, "view")
+        networks_table = CloudNetworkTable(
+            data=networks,
+            extra_columns=[("actions", ButtonsColumn(model=CloudNetwork, return_url_extra="?tab=networks"))],
+        )
+        networks_table.columns.hide("cloud_resource_type")
+        RequestConfig(
+            request, paginate={"paginator_class": EnhancedPaginator, "per_page": get_paginate_count(request)}
+        ).configure(networks_table)
+        network_url = reverse("cloud:cloudnetwork_add")
+        network_table_add_url = f"{network_url}?cloud_resource_type={instance.id}"
+        return Response(
+            {"networks_table": networks_table, "active_tab": "networks", "network_table_add_url": network_table_add_url}
+        )
 
-        return context
+    @action(detail=True, url_path="services", url_name="services")
+    def services(self, request, *args, **kwargs):
+        instance = self.get_object()
+        services = instance.cloud_services.restrict(request.user, "view")
+        services_table = CloudServiceTable(
+            data=services,
+            extra_columns=[("actions", ButtonsColumn(model=CloudService, return_url_extra="?tab=services"))],
+        )
+        services_table.columns.hide("cloud_resource_type")
+        RequestConfig(
+            request, paginate={"paginator_class": EnhancedPaginator, "per_page": get_paginate_count(request)}
+        ).configure(services_table)
+        service_url = reverse("cloud:cloudservice_add")
+        service_table_add_url = f"{service_url}?cloud_resource_type={instance.id}"
+
+        return Response(
+            {
+                "services_table": services_table,
+                "active_tab": "cloudservices",
```

Comment:
Should this be since the tab_id on line 154 is "services" instead of "cloudservices"?
```suggestion
                "active_tab": "services",
```

Reply:
Updated as requested, should be resolve, let me know if I missed anything